CRM/Price/BAO/PriceSet.php

```diff
@@ -673,13 +673,15 @@ public static function processAmount($fields, &$params, &$lineItem, $priceSetID
         continue;
       }
 
-      list($params, $lineItem, $totalTax, $totalPrice) = self::getLine($params, $lineItem, $priceSetID, $field, $id, $totalPrice);
+      list($params, $lineItem) = self::getLine($params, $lineItem, $priceSetID, $field, $id, $totalPrice);
     }
 
     $amount_level = [];
     $totalParticipant = 0;
     if (is_array($lineItem)) {
```

Comment:
I like how all the math is starting to be consolidated.
Maybe move `$totalParticipant = 0;` up to L667 -> so that we set all 
`$totalPrice = $totalTax = $totalParticipant = 0;`

Reply:
@KarinG yes - makes sense - I'd rather leave it for follow up now if this works as it will be a few days before I get more time